applications/CompressiblePotentialFlowApplication/tests/naca0012_small_compressible_test/moment.dat

```diff
@@ -0,0 +1 @@
+-0.114870305288
```

Comment:
Same comment.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668